python/eos/plot/plotter.py

```diff
@@ -361,20 +361,22 @@ class Observable(BasePlot):
            }
         """)
 
-        from .specs import Specs, ParametersSpec, KinematicsSpec,\
-        RangeSpec, ObservableSpec, VariableSpec, ParameterFileSpec
+        # from .specs import SpecCollection, ParametersSpec, KinematicsSpec,\
+        # RangeSpec, ObservableSpec, VariableSpec, ParameterFileSpec
+        from .specs import SpecCollection, ObservableSpec, VariableSpec
 
-        specs_prototype = Specs([
+        specs_prototype = SpecCollection([
```

Comment:
Do we need to mark this as private and thus let the interpreter mangle it? I think mangling is needed here.
(Also, can we get rid of ``_prototype``?)

```python
class BasePlot:
    self.specs = deepcopy(BasePlot._specs)

class Observable:
    self.specs += deepcopy(Observable._specs)

Reply:
Do I understand this correctly that you suggest the following?

```python3
class BasePlot:
    _specs = ... # class member

    __init__(self):
       self.specs = deepcopy(BasePlot._specs) # instance member
```

It is supposed to be accessible from the outside for the documentation, though. I would consider this as api.
If python would allow it, the class member should be const whereas the copies would be mutable. But all we can do is somewhat mark this via `_prototype` or similar.